src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/Runners/AppRunner.cs

```diff
@@ -60,6 +60,8 @@ public sealed class AppRunner : IAsyncDisposable
 
         public int ProcessId => _adapter.ProcessId;
```

Comment:
Do we still use this property? Or would you expect all access to be done through the task?

Reply:
There are 33 uses of this property still. I can try to go through all of them and convert them to using the task.

Reply:
I've removed the property and replaced with with accesses to the ProcessIdTask property.